mne/viz/evoked.py

```diff
@@ -1440,21 +1442,170 @@ def _truncate_yaxis(axes, ymin, ymax, orig_ymin, orig_ymax, fraction,
     return ymin_bound, ymax_bound
 
 
-def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
-                         linestyles=['-'], styles=None, vlines=list((0.,)),
-                         ci=0.95, truncate_yaxis=False, truncate_xaxis=True,
-                         ylim=dict(), invert_y=False, show_sensors=None,
-                         show_legend=True, axes=None, title=None, show=True):
+def _combine_grad(evoked, picks):
+    """Creates a new instance of Evoked with combined gradiometers (RMSE)"""
+    def pair_and_combine(data):
+        data = data ** 2
+        data = (data[::2, :] + data[1::2, :]) / 2
+        return np.sqrt(data)
+    picks, ch_names = _grad_pair_pick_and_name(evoked.info, picks)
+    this_data = pair_and_combine(evoked.data[picks, :])
+    ch_names = ch_names[::2]
+    evoked = evoked.copy().pick_channels(ch_names)
+    combined_ch_names = [ch_name[:-1] + "X" for ch_name in ch_names]
+    evoked.rename_channels({c_old: c_new for c_old, c_new
+                            in zip(evoked.ch_names, combined_ch_names)})
+    evoked.data = this_data
+    return evoked
+
+
+def _check_loc_legal(loc, what='your choice'):
+    """Check if a loc is a legal for MPL."""
```

Comment:
"""Check if loc is a legal location for MPL subordinate axes."""
(or something like that, it's missing a noun)